vizro-mcp/README.md

```diff
@@ -0,0 +1,193 @@
+      "command": "uvx",
+      "args": [
+        "vizro-mcp"
+      ]
+    }
+  }
+}
+```
+
+> ⚠️ **Warning:** In some cases you may need to provide the full path to your `uv` executable, so instead of `uv` would use something like `/Users/<your-username>/.local/bin/uv`. To discover the path of `uv` on your machine, in your terminal app, type `which uv`.
+
+Similarly, when using Cursor, after a short pause, you should see a green light in the MCP menu:
+
+<img src="assets/cursor_working.png" alt="Cursor MCP Server Icon" width="400"/>
+
+</details>
+
+In principle, the Vizro MCP server works with _any_ MCP enabled LLM applications but we recommend Claude Desktop or Cursor as popular choices. Different AI tools may use different setup methods or connection settings. Check each tool's docs for details.
+
+## 💻 Usage
+
+### Use prompt templates to get specific dashboards quickly
+
+Prompt templates are not available in all MCP hosts, but when they are, you can use them to get specific dashboards quickly. To access them (e.g. in Claude Desktop), click on the plus icon below the chat, and choose _`Add from vizro-mcp`_.
+
+<img src="assets/claude_prompt.png" alt="Claude Desktop MCP Server Icon" width="300"/>
+
+The **easiest** way to get started with Vizro dashboards is to choose the template `create_starter_dashboard` and just send the prompt. This will create a super simple dashboard with one page, one chart, and one filter. Take it from there!
+
+### Create a Vizro dashboard based on local or remote data
+
+You can also ask the LLM to create specific dashboards based on local or remote data if you already have an idea of what you want. Example prompts could be:
+
+> _Create a Vizro dashboard with one page, a scatter chart, and a filter based on `<insert absolute file path or public URL>` data._
+
+> _Create a simple two page Vizro dashboard, with first page being a correlation analysis of `<insert absolute file path or public URL>` data, and the second page being a map plot of `<insert absolute file path or public URL>` data_
+
+You can even ask for a dashboard without providing data:
+
+> _Create a Vizro dashboard with one page, a scatter chart, and a filter._
+
+In general, it helps to specify Vizro in the prompt and to keep it as precise (and simple) as possible.
+
+### Get a live preview of your dashboard
+
+When the LLM chooses to use the tool `validate_model_config`, and the tool executes successfully, the LLM will return a link to a live preview of the dashboard if only public data accessed via URL is used. By default, the LLM will even open the link in your browser for you unless you tell it not to. In Claude Desktop, you can see the output of the tool by opening the tool collapsible and scrolling down to the very bottom.
+
+<img src="assets/claude_validate.png" width="300"/>
+
+You can also ask the model to give you the link, but it will attempt to regenerate it, which is very error prone and slow.
+
+### Create Vizro charts
```

Comment:
I think it would help new users to say what the difference is between a chart and a dashboard. It's so obvious to us but it bears telling that sometimes you may just want the code for a single chart (and you could point at the visual vocabulary for good measure to illustrate some of the examples of chart code we provide).